astropy/io/fits/tests/test_fitstime.py

```diff
@@ -129,6 +129,7 @@ def test_fits_to_time_meta(self, table_types):
         t.meta['MJD-OBS'] = 56670
 
         # Test for default read/write behaviour (raw data)
```

Comment:
This comment should be adjusted. Maybe just "Test for conversion of time data to its `value`, as defined by its format" or something like that.

As it is *not* the default, I would also suggest to swap the test, so that the default is tested first (but this is really a very minor detail).